layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -15,4 +38,17 @@ SRC_URI:append:beaglebone-ai64 = " \
 # The BB-AI64 creates this file for both archs, remove one.
 do_deploy:append:beaglebone-ai64 () {
     rm -vf ${DEPLOYDIR}/extra_uEnv.txt
-}
\ No newline at end of file
+}
+
+# The Beagleplay *also* creates this file for both archs, remove one.
+do_deploy:append:beagleplay () {
+    rm -vf ${DEPLOYDIR}/extra_uEnv.txt
+}
+
+SSTATE_ALLOW_OVERLAP_FILES += "${DEPLOYDIR}/extra_uEnv.txt"
```

Comment:
Why is this needed? I don't think there is another recipe providing this file

Reply:
I was seeing errors before I discovered the following trick (from whoever added the BB-AI64 support):

```
# The Beagleplay *also* creates this file for both archs, remove one.
do_deploy:append:beagleplay () {
    rm -vf ${DEPLOYDIR}/extra_uEnv.txt
}
```

I think we can remove this line. Will rebuild and retest.